app/views/applicants/show.html.erb

```diff
@@ -5,13 +5,15 @@
 <h3><%= @applicant.zipcode %></h3>
 <h3><%= @applicant.description %></h3>
 
-<%= form_with url: "/applicants/#{@applicant.id}", method: :get, local: true do |f| %>
-  <%= f.label :search, 'Search pets to adopt:' %>
-  <%= f.text_field :search %>
-  <%= f.submit 'Search' %>
+<% if @applicant.status == 'In Progress' %>
+  <%= form_with url: "/applicants/#{@applicant.id}", method: :get, local: true do |f| %>
+    <%= f.label :search, 'Search pets to adopt:' %>
+    <%= f.text_field :search %>
+    <%= f.submit 'Search' %>
+  <% end %>
 <% end %>
 
-<% if @pets != nil %>
+<% if @pets.present? %>
```

Comment:
This change was consistent with the ones I made in the last refactor, thank you!